### PR TITLE
Add neuroglancer and volview

### LINKED ISSUE #264: Add more viewers
Add these
* https://imjoy.io/#/app?plugin=https%3A%2F%2Fkitware.github.io%2Fitk-vtk-viewer%2Fapp%2F
* https://imjoy.io/lite?plugin=https://kitware.github.io/itk-vtk-viewer/app/
* https://kitware.github.io/itk-vtk-viewer/app/?fileToLoad=https://data.kitware.com/api/v1/file/564a65d58d777f7522dbfb61/download/data.nrrd
* neuroglancer?

### PR DESCRIPTION
**Pull request by SeanLeRoy**
## Description
Yesterday during the data summit there was a lot of interest in getting Neuroglancer support in BFF so this adds Neuroglancer & VolView (just for fun). This also makes it so that apps that aren't expected to support the file get brought into an "Other" category at the bottom. I did not make the 3D Web Viewer, AGAVE, or Neuroglancer file type specific because .zarr files often don't have an extension and so these kind of act like defaults - we might want to consider changing that up later.

## Related Issue
Resolves #264

### CODE REVIEW

File: packages/core/hooks/useOpenWithMenuItems/index.tsx [hunk 1]
```diff
@@ -9,6 +9,90 @@ import { interaction, metadata } from "../../state";
...
+        title: `Open files with Neuroglancer`,
+        href: `https://neuroglancer-demo.appspot.com/#!{%22layers%22:[{%22source%22:%22zarr://${fileDetails?.cloudPath}%22,%22name%22:%22${fileDetails?.name}%22}]}`,
+        disabled: !fileDetails?.path,
+        target: "_blank",
+    },
+    simularium: {
+        key: "simularium",
+        text: "Simularium",
+        title: `Open files with Simularium`,
+        href: `https://simularium.allencell.org/viewer?trajUrl=${fileDetails?.cloudPath}`,
```
Comment (aswallace) [inline packages/core/hooks/useOpenWithMenuItems/index.tsx:40]: should this one also be disabled on `!fileDetails?path` or not an issue for this one?

**Review by aswallace — APPROVED** [review]
Do we currently have simularium etc files in aics that we can try this out on?

File: packages/core/hooks/useOpenWithMenuItems/index.tsx (hunk 1 above)
Comment (SeanLeRoy) [reply packages/core/hooks/useOpenWithMenuItems/index.tsx:40]: Ah nope it should I missed that

**Comment by SeanLeRoy** [comment]
> Do we currently have simularium etc files in aics that we can try this out on?

Not really, you have to download them from Simularium's website

File: packages/core/hooks/useOpenWithMenuItems/index.tsx (hunk 1 above)
Comment (SeanLeRoy) [reply packages/core/hooks/useOpenWithMenuItems/index.tsx:40]: Done in https://github.com/AllenInstitute/biofile-finder/pull/281/commits/6fc3624ebf1c7f9ccff8ebbcef6c0fab6100681b

**Review by BrianWhitneyAI — APPROVED** [review]

**Pull request merged**